website/source/api/config.html.md

```diff
@@ -0,0 +1,246 @@
+| `PUT`  | `/config`                    | `application/json`         |
+
+The table below shows this endpoint's support for
+[blocking queries](/api/index.html#blocking-queries),
+[consistency modes](/api/index.html#consistency-modes),
+[agent caching](/api/index.html#agent-caching), and
+[required ACLs](/api/index.html#acls).
+
+| Blocking Queries | Consistency Modes | Agent Caching | ACL Required  |
+| ---------------- | ----------------- | ------------- | ------------- |
+| `NO`             | `none`            | `none`        | `service:write`<br>`operator:write`<sup>1</sup> |
+
+<sup>1</sup> The ACL required depends on the Kind of the config entry being updated:
+
+| Config Entry Kind | Required ACL     |
+| ----------------- | ---------------- |
+| service-defaults  | `service:write`  |
+| proxy-defaults    | `operator:write` |
+
+### Parameters
+
+- `dc` `(string: "")` - Specifies the datacenter to query. This will default to
+  the datacenter of the agent being queried. This is specified as part of the
+  URL as a query parameter.
+
+- `cas` `(int: 0)` - Specifies to use a Check-And-Set operation. If the index is 
+  0, Consul will only put the entry if it does not already exist. If the index is 
+  non-zero, the entry is only set if the current index matches the `ModifyIndex` 
+  of that entry.
+
+### Sample Payload
+
+
+```javascript
+{
+    "Kind": "service-defaults",
+    "Name": "web",
+    "Protocol": "http",
+    "Connect": {
+        "SidecarProxy": false
+    }
+}
+```
+
+### Sample Request
+
+```text
+$ curl \
+    --request PUT \
+    --data @payload \
+    http://127.0.0.1:8500/v1/config
+```
+
+## Get Configuration
+
+This endpoint returns a specific config entry.
+
+| Method | Path                         | Produces                   |
+| ------ | ---------------------------- | -------------------------- |
+| `GET`  | `/config/:kind/:name`        | `application/json`         |
+
+The table below shows this endpoint's support for
+[blocking queries](/api/index.html#blocking-queries),
+[consistency modes](/api/index.html#consistency-modes),
+[agent caching](/api/index.html#agent-caching), and
+[required ACLs](/api/index.html#acls).
+
+| Blocking Queries | Consistency Modes | Agent Caching | ACL Required  |
+| ---------------- | ----------------- | ------------- | ------------- |
+| `YES`            | `all`             | `none`        | `service:read`<sup>1</sup> |
+
+<sup>1</sup> The ACL required depends on the Kind of the config entry being read:
+
+| Config Entry Kind | Required ACL     |
+| ----------------- | ---------------- |
+| service-defaults  | `service:read`  |
+| proxy-defaults    | `<none>` |
+
+### Parameters
+
+- `dc` `(string: "")` - Specifies the datacenter to query. This will default to
+  the datacenter of the agent being queried. This is specified as part of the
+  URL as a query parameter.
+
+- `kind` `(string: <required>)` - Specifies the Kind of the entry to read. This
```

Comment:
```suggestion
- `kind` `(string: <required>)` - Specifies the kind of the entry to read. This
```

Reply:
Similar to another comment above. The capitalization I think was to indicate that this is a field on the structure that is sent to the API and stored in Raft.

Should we not do that? Or maybe we should wrap it in backticks to stand out more as being special and not just a typo?

Reply:
we don't actually pass it as a field in a structure here so it seemed more consistent with everywhere else to lowercase it.